ftpvl/evaluation.py

```diff
@@ -47,3 +53,37 @@ def process(self, pipeline: List['Processor']) -> 'Evaluation':
             an Evaluation instance that was processed by the pipeline
         """
         return reduce(lambda r, p: p.process(r), pipeline, self)
+
+    def __add__(self, other: 'Evaluation') -> 'Evaluation':
+        """
+        Magic method for concatenating two Evaluations, returning an Evaluation
+        with dataframe (self + other).
+
+        Args
+        ------
+            other: the other Evaluation to concatenate
+        
+        Returns:
+            a new Evaluation that consists of the two Evaluations concatenated
+        """
+        if not isinstance(other, Evaluation):
+            raise TypeError(f"can only concatenate Evaluation (not {type(other).__name__}) to Evaluation")
+        new_df = pd.concat([self.get_df(), other.get_df()], ignore_index=True)
+        return Evaluation(new_df)
+
+    def __radd__(self, other: 'Evaluation') -> 'Evaluation':
```

Comment:
Great question. `__radd__` needs to be implemented so that `sum()` works. By default, the start value that `sum` uses is `0`, so for `sum([Evaluation(1), Evaluation(2)])` the first operation is `0 + Evaluation(1)`. This would call `__radd__`, and to handle this gracefully, we need to check if `other == 0`, and if so, ignore it (returning self, which effectively sets the first evaluation as the start value).

This means technically line 89 isn't necessary and we can just throw an error there, but instead I have it handled in `__add__` to avoid repetitiveness.